packages/design-system/src/components/InputField/style.scss

```diff
@@ -11,6 +11,7 @@
   transition: border 0.25s ease-in-out;
   will-change: border-color;
   background-clip: padding-box;
+  width: 100%;
```

Comment:
I feel that is a right thing to do because everyone pass width: 100% there anyway but we would need to use the same value for i.e textarea to make it consistent